added executor_threads documentation (#421)
logstash-plugins/logstash-input-beats: docs/index.asciidoc

Close Idle clients after X seconds of inactivity.

[id="plugins-{type}s-{plugin}-executor_threads"]
===== `executor_threads`

  * Value type is <<number,number>>
  * Default value is 1 executor thread per CPU core

> The number of parallel executor threads.

Reviewer: ```suggestion
The number of parallel executor threads to be used to process incoming beats requests
```

Refer to <<plugins-{type}s-{plugin}-ecs_metadata,ECS mapping>> for detailed information.

[id="plugins-{type}s-{plugin}-executor_threads"]
===== `executor_threads`

  * Value type is <<number,number>>
  * Default value is 1 executor thread per CPU core

The number of threads to be used to process incoming beats requests.

> By default Beats input will create a number of threads equals to 2*CPU cores to handle incoming connections,
> reading from the established sockets and execute most of the tasks related to network connection managements, 
> except the parsing of Lumberjack protocol that's offloaded to a dedicated thread pool.

Reviewer 2: ```suggestion
By default the Beats input creates a number of threads equal to 2*CPU cores.
These threads handle incoming connections, reading from established sockets, and executing most of the tasks related to network connection management. 
Parsing the Lumberjack protocol is offloaded to a dedicated thread pool.
```

Author: Nice, thank you Karen for the additional suggestions.
